From 883cfd41ab4b39528ba94f272bd2d78e009aa92d Mon Sep 17 00:00:00 2001
Message-Id: <883cfd41ab4b39528ba94f272bd2d78e009aa92d@dist-git>
From: Jiri Denemark <jdenemar@redhat.com>
Date: Tue, 26 Feb 2019 12:22:00 +0100
Subject: [PATCH] cpu_map: Add more signatures for IvyBridge CPU models
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
Reviewed-by: Ján Tomko <jtomko@redhat.com>
(cherry picked from commit 194105fef1a3a8645486df3323e460cc4a9b2d4c)

https://bugzilla.redhat.com/show_bug.cgi?id=1558558
https://bugzilla.redhat.com/show_bug.cgi?id=1687515

Conflicts:
	src/cpu_map/x86_IvyBridge-IBRS.xml
	src/cpu_map/x86_IvyBridge.xml
            - cpu_map split not backported

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
Reviewed-by: Ján Tomko <jtomko@redhat.com>
---
 src/cpu/cpu_map.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/cpu/cpu_map.xml b/src/cpu/cpu_map.xml
index 8fe6d7d4f4..68cecf7ac0 100644
--- a/src/cpu/cpu_map.xml
+++ b/src/cpu/cpu_map.xml
@@ -1103,6 +1103,7 @@
 
     <model name='IvyBridge'>
       <signature family='6' model='58'/> <!-- 0306a0 -->
+      <signature family='6' model='62'/> <!-- 0306e0 -->
       <vendor name='Intel'/>
       <feature name='aes'/>
       <feature name='apic'/>
@@ -1152,6 +1153,7 @@
 
     <model name='IvyBridge-IBRS'>
       <signature family='6' model='58'/> <!-- 0306a0 -->
+      <signature family='6' model='62'/> <!-- 0306e0 -->
       <vendor name='Intel'/>
       <feature name='aes'/>
       <feature name='apic'/>
-- 
2.21.0

